kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/DisbursementVoucherNonEmployeeTravelValidation.java

```diff
@@ -0,0 +1,214 @@
+package edu.arizona.kfs.fp.document.validation.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherNonEmployeeTravel;
+import org.kuali.kfs.fp.document.DisbursementVoucherConstants;
+import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
+import org.kuali.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSKeyConstants;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.document.validation.event.AttributedDocumentEvent;
+import org.kuali.rice.core.api.parameter.ParameterEvaluator;
+import org.kuali.rice.core.api.parameter.ParameterEvaluatorService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+import org.kuali.rice.krad.util.GlobalVariables;
+import org.kuali.rice.krad.util.MessageMap;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public class DisbursementVoucherNonEmployeeTravelValidation extends org.kuali.kfs.fp.document.validation.impl.DisbursementVoucherNonEmployeeTravelValidation {
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(DisbursementVoucherNonEmployeeTravelValidation.class);
+	private ParameterService parameterService;
+
+	public void setParameterService(ParameterService parameterService) {
+		this.parameterService = parameterService;
+	}
+
+	@Override
+	public boolean validate(AttributedDocumentEvent event) {
+		LOG.debug("validate start");
+		boolean isValid = true;
+
+		DisbursementVoucherDocument document = (DisbursementVoucherDocument) getAccountingDocumentForValidation();
+		DisbursementVoucherNonEmployeeTravel nonEmployeeTravel = document.getDvNonEmployeeTravel();
+
+		// skip the validation if the payment reason is not noneployee travel or
+		// the payee is an employee
+		if (!isTravelNonEmplPaymentReason(document)) {
+			return true;
+		}
+
+		MessageMap errors = GlobalVariables.getMessageMap();
+		errors.addToErrorPath(KFSPropertyConstants.DOCUMENT);
+		errors.addToErrorPath(KFSPropertyConstants.DV_NON_EMPLOYEE_TRAVEL);
+
+		getDictionaryValidationService().validateBusinessObjectsRecursively(document.getDvNonEmployeeTravel(), 1);
+		
+		LOG.info("Errors?" + errors.toString());
+		if (ObjectUtils.isNull(errors)) {
```

Comment:
Is the LOG level of 'info' correct for these messages around "errors"? This looks to be more along the lines of developer debug logging. Can this be removed or are these statements beneficial for a production environment?